packages/@o3r/core/schematics/ng-update/index.ts

```diff
@@ -34,7 +34,8 @@ function updateV10_0Fn(): Rule {
     const updateRules: Rule[] = [
       // Some of these imports were missed in the generators of v9, so it's easier to just run the update again
       updateImports(mapImportAsyncStore),
-      updateConfiguration
+      updateConfiguration,
+      alignVersions([/@o3r\/.*/, /@ama-sdk\/.*/])
```

Comment:
aligning only otter dependencies won't be enough because of the transitive dependencies (angular, ngrx, ...)
we'll need a full version-harmonize